python/paddle/distributed/fleet/meta_parallel/pipeline_parallel.py

```diff
@@ -1186,3 +1190,227 @@ def eval_batch(self, data, compute_loss=False):
         self._compute_loss = compute_loss
 
         return self.forward_backward_pipeline(data, None, forward_only=True)
+
+
+class PipelineParallelWithInterleaveFthenB(PipelineParallelWithInterleave):
+    def __init__(self, layers, hcg, strategy):
```

Comment:
Add some comments to explain why this is done and how it differs from `PipelineParallelWithInterleave`.

Reply:
OK，i will fix in next pr.